src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresViewIT.java

```diff
@@ -100,56 +110,81 @@ public void testRecordSchemaMoreFieldsThanViewFails() throws SQLException {
         .put("firstname", "Christina")
         .put("lastname", "Brams")
         .put("age", 20);
-    try {
-      task.put(Collections.singleton(new SinkRecord(topic, 1, null, null, schema, struct, 1)));
-      fail();
-    } catch (ConnectException e) {
-      assertTrue(e.getMessage().contains("View \"" + topic + "\" is missing fields"));
-    }
+
+    String kafkaValue = new String(jsonConverter.fromConnectData(topic, schema, struct));
+    connect.kafka().produce(topic, null, kafkaValue);
+
+    ConsumerRecords<byte[], byte[]> records = connect.kafka().consume(1, CONSUME_MAX_DURATION_MS, DLQ_TOPIC_NAME);
+
+    assertEquals(1, records.count());
   }
 
   @Test
-  public void testRecordSchemaLessFieldsThanView() throws SQLException {
+  public void testRecordSchemaLessFieldsThanView() throws Exception {
+    connect.configureConnector("jdbc-sink-connector", props);
+    waitForConnectorToStart("jdbc-sink-connector", 1);
+
```

Comment:
I think we should preserve the order of creating the table and view before starting connector.